source/_components/cppm.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: "Aruba ClearPass"
+description: "Instructions on how to integrate Aruba routers into Home Assistant."
+date: 2019-03-05 10:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: aruba.png
+ha_category: Presence Detection
+ha_release: 0.89
+---
+
+
+This platform allows you to detect presence by looking at connected devices to [Aruba Clearpass](https://www.arubanetworks.com/products/security/network-access-control/).
+
+Supported platforms (tested):
+
+- Aruba ClearPass 6.7.5
+
+<p class='note warning'>
+You must first creat an API client from here: https://clearpass.server.com/guest/api_clients.php
+</p>
+
+To use this device tracker in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: cppm_tracker
+    host: clearpass.server.org
+    client_id: clearpassapi
+    api_key: 00000004qyO513hTdCfjIO2ZWWnmex8QZ5000000000
+    scan_interval: 120
```

Comment:
Just keep the required config items in the first example. More options can be shown in a second example further below.